components/Header.tsx

```diff
@@ -1,15 +1,18 @@
 import Image from "next/image";
 import Link from "next/link";
 import { useRouter } from "next/router";
-
 import { useAppData } from "../hooks/useAppData";
 import { Tag } from "../types";
 
+const getActiveTagOrLanguage = (tag, language, languages, tags) => {
```

Comment:
I believe there is another function somewhere that does the same? Sorry, I lost track of all PRs ;)

Reply:
As far as i have seen the codebase, i didn't find anything similar to this function.